src/material/list/list-option.html

```diff
@@ -4,21 +4,25 @@
   into a template since we use it in multiple places.
 -->
 <ng-template #icons>
-  <ng-content select="[matListItemAvatar],[matListItemIcon]">
-  </ng-content>
+  <ng-content select="[matListItemAvatar],[matListItemIcon]"> </ng-content>
 </ng-template>
 
 <ng-template #checkbox>
   <div class="mdc-checkbox" [class.mdc-checkbox--disabled]="disabled">
-    <input type="checkbox" class="mdc-checkbox__native-control"
-           [checked]="selected" [disabled]="disabled"/>
+    <input
+      type="checkbox"
+      class="mdc-checkbox__native-control"
+      [checked]="selected"
+      [disabled]="disabled"
+      [attr.aria-labelledby]="labelId"
```

Comment:
This input doesn't need to be labelled, because it's `display: none` so it's completely removed from the accessibility tree. See https://github.com/angular/components/blob/main/src/material/list/list-option.scss#L76